### PR TITLE
Use prebuilt docker images

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

**Review by as2388 — APPROVED** [review]
Thanks, please let us know what testing you have done to confirm that the results are the same before and after using the docker image

File: configurations/docker_image_name.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ghcr.io/africasvoices/engagement-data-pipeline:match-values-in-coda-sync
```
Comment (IsaackMwenda) [inline configurations/docker_image_name.txt]: we now have v1.2.0

**Review by IsaackMwenda — APPROVED** [review]
LGTM - pending before and after analysis files comparison.

**Comment by DanielMwendwa** [comment]
> the results are the same before and after using the docker image

I had compared the analysis files to what we had before and there are the same.

**Pull request merged**